### DISCUSSION TITLE
「开源之史」系列之十五/四：君子协定 —— 联盟以对抗万恶的专利 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#73] [Announcements]
# 「开源之史」系列之十五/四：君子协定 —— 联盟以对抗万恶的专利 - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-11-os-and-software-patents/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

专利以及专利蟑螂的存在本身就已经在证明, 专利本身已经背弃了自己的初衷;

原本想用来鼓励创新的制度, 已经现实上在劝退创新;
因为, 当前的创新已经不是一个独立组件或是产品就可以占领一个市场, 
完成创新的回利了;

世界从未如此紧密的纠缠在一起, 任何一个领域的创新, 都必须依赖越来越多其它领域的创新;

而专利, 特别是创新专利, 根本不用给出实体嘦提出想法就可以受到保护,
当然,每年要交纳保护费/专利费;
结果专利就变成了一种生意, 
专利池就变成了企业创新的护城河, 
一切产业化后, 专利变成了核武器, 只有相互制约时, 才有用,
否则, 无处可用...

怎么办? Tesla 已经指出了方向,
和 GPL 一样, 用专利来反对专利, 
将领域所有专利公开, 但是, 用无解的生产效能来把住市场, 以及给自己合理的压力,
这当然是勇敢者游戏,
如果没有开创一个全新领域, 以及对应成套专利的话, 也玩不起来,
好在开源方面的积累足够,
唯一问题在每年的保护费...
